versions/next/asyncapi.md

```diff
@@ -47,7 +47,7 @@ Means that the [application](#definitionsApplication) is a [consumer](#definitio
       - [Info Object](#infoObject)
       - [Contact Object](#contactObject)
       - [License Object](#licenseObject)
-      - [Servers Object](#A2SServers)
+      - [Servers Object](#serverObject)
```

Comment:
This link is not correct. It should link to the "servers object" but it's missing. We should create the Servers Object section in the spec.

Reply:
Oh, I see. But wouldn't the "servers object" just be an array of server objects?

Reply:
Yes, that's why it was linked to `#A2SServers`. Ideally and since every object has its own "space" in the spec, this should too. It would be a matter of adding another section saying it's just an array of Server Objects.

What do you think?

Reply:
Okay, makes sense. I can put that in. I realize that the task to put in the Servers Object is in #192 , I could do that first.

Reply:
I got back to working on this. I wrote a tiny section called Servers Object and said it was just an array of Server Objects. I'm having trouble seeing the value in this. The AsyncAPI object has a field named servers which is an array of Server Objects, rather than a Servers Object. Creating that extra bit of doc seems to me to just be adding noise. Maybe I'm misunderstanding something!

Reply:
Well, it's actually just to be consistent. Think this is a specification, not a documentation page or article. Sometimes you have to write obvious things to make them explicit.

Reply:
Okay, I'll carry on then.